java/src/jmri/jmrit/display/layoutEditor/LayoutEditorAuxTools.java

```diff
@@ -416,7 +416,7 @@ else if (lc.getConnectedType() >= LayoutTrack.SLIP_A
             // block2 is this LayoutBlock, and block1 is in a track segment
             if (lc.getConnectedObject() != null) {
                 // connected object in this block is a turnout or levelxing
-                curConnection = lc.getConnectedObject();
+                curConnection = (LayoutTrack) lc.getConnectedObject();
```

Comment:
I think this cast is also unneeded.

Reply:
Done. Also changed some Objects to LayoutTrack's.